### DISCUSSION TITLE
Snaking out sliders does not work with No slider head movement

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16872] [Q&A] [answered]
Without Classic:

https://user-images.githubusercontent.com/64581009/153882087-9164630b-f202-4714-8bc5-de9439ed85e5.mp4

With Classic:

https://user-images.githubusercontent.com/64581009/153882118-d666631c-152d-4974-a707-c966cd37b670.mp4

A bit of an inconsistency there. Feels weird to look at when playing with classic. I assume the option is tied to the actual slider head's movement itself.

**Comment by peppy** [comment, marked as answer]
It shows the actual judgement range.

**Reply by NotGumballer91** [reply]
Yes, I'm aware. What I'm saying is that it doesn't feel right if the slider doesn't snake out entirely. Most players expects it to be snaking out than being technically accurate.

**Reply by peppy** [reply]
Wait I'm not sure what you're after. Snaking out was not a thing on classic (stable) so the fact it's disabled is probably intentional?

**Reply by frenzibyte** [reply]
Moreover, this is touched on in the logic which forcefully disables it:

https://github.com/ppy/osu/blob/4f9c3fde07a662ef43925aadb4cd562c2afed1ac/osu.Game.Rulesets.Osu/Skinning/Default/PlaySliderBody.cs#L57-L62

You can't have the slider snaking out when the head is pinned to its location.
